backend/sublet/serializers.py

```diff
@@ -110,8 +150,34 @@ def destroy(self, instance):
             raise serializers.ValidationError("You do not have permission to delete this sublet.")
 
 
+class SubletSerializerRead(serializers.ModelSerializer):
+    amenities = AmenitySerializer(many=True, required=False)
+    images = SubletImageURLSerializer(many=True, required=False)
+
+    class Meta:
+        model = Sublet
+        read_only_fields = ["id", "created_at", "subletter", "sublettees"]
+        fields = [
+            "id",
+            "subletter",
+            "amenities",
+            "title",
+            "address",
+            "beds",
+            "baths",
+            "description",
+            "external_link",
+            "min_price",
+            "max_price",
+            "start_date",
+            "end_date",
+            "expires_at",
+            "images",
+        ]
+
+
 # simple sublet serializer for use when pulling all serializers/etc
-class SimpleSubletSerializer(serializers.ModelSerializer):
+class SubletSerializerSimple(serializers.ModelSerializer):
```

Comment:
Do we still need this serializer now that we have `SubletSerializerRead`?  Seems like the only difference is the read only fields, but I don't think that matters after looking at the simpler serializer's use cases.

Reply:
Good point; the original idea of the simple version is to send only the data necessary for a browse view (as opposed to viewing a singular sublet), but not sure if the few text fields being saved are worth. Thoughts @vcai122?

Reply:
discussed - worth leaving since we already made it